doc/source/guide/data_types/timeseries.rst

```diff
@@ -0,0 +1,378 @@
+
+    >>> from sunpy.util.metadata import MetaDict
+    >>> from collections import OrderedDict
+    >>> import astropy.units as u
+    >>> meta = MetaDict({'key':'value'})
+    >>> units = OrderedDict([('intensity', u.W/u.m**2)])
+    >>> ts_table = sunpy.timeseries.TimeSeries(table, meta, units)
+
+
+3. Inspecting TimeSeries & Getting at the Data
+-----------------------------------------------
+
+A time series holds both data as well as meta data and units data. The meta data for the time series is accessed by: ::
+
+    >>> header = my_timeseries.meta
+
+This references the `~sunpy.timeseries.metadata.TimeSeriesMetaData` object with the header information as read
+from the source files.
+A word of caution, many data sources provide little to no meta data so this variable might be empty.
+The meta data is described in more detail later in this guide.
+Similarly there are properties for getting `~sunpy.timeseries.timeseriesbase.GenericTimeSeries.columns` as a list of strings, `~sunpy.timeseries.timeseriesbase.GenericTimeSeries.index` values and `~sunpy.timeseries.timeseriesbase.GenericTimeSeries.time_range` of the data.
+The actual data in a SunPy TimeSeries object is accessible through the
+`~sunpy.timeseries.timeseriesbase.GenericTimeSeries.data` attribute.  The data is implemented as a
+Pandas `~pandas.DataFrame`, so to get a look at what data you have available use: ::
+
+    >>> my_timeseries.data
+
+You can also get a quick overview of that data using: ::
+
+    >>> my_timeseries.data.info()
+
+Time series are columnar data so to get at a particular datum you need to
+first index the column then the element you want. To get the names of the
+available columns: ::
+
+    >>> my_timeseries.data.columns
+
+You can access the 0th element in the column `xrsa` with: ::
+
+    >>> my_timeseries.data['xrsa'][0]
+
+You can also grab all of the data at a particular time: ::
+
+    >>> my_timeseries.data['xrsa']['2012-06-01 00:00']
+
+This will return a list of entries with times that match the accuracy of the time
+you provide. You can consider the data as x or y values: ::
+
+    >>> x = my_timeseries.data.index
+    >>> y = my_timeseries.data.values
+
+You can read more about indexing at the `pandas documentation website
+<http://pandas.pydata.org/pandas-docs/stable/>`_.
+
+A TimeSeries can also return an AstroPy `~astropy.units.quantity.Quantity` for a given column using the `~sunpy.timeseries.timeseriesbase.GenericTimeSeries.quantity` method, this uses the values stored in the data and units stored in the units dictionary to determine the `~astropy.units.quantity.Quantity`: ::
+
+    >>> quantity = my_timeseries.quantity('xrsa')
+
+4. Plotting
+-----------
+
+The SunPy TimeSeries object has its own built-in plot methods so that
+it is easy to quickly view your time series. To create a plot just
+type: ::
+
+.. plot::
+    :include-source:
+    
+    my_timeseries.peek()
```

Comment:
does this actually work?! I thought the `plot` directive needed a complete code example?